8 add dashboard to wiki (#560)
iotaledger/integration-services: documentation/docs/services/dashboard.md

---
image: /img/integration-services/logo/integration_services.png
description: The dashboard is part of the Integration Services Suite and enables management of identities and streams.  
keywords:
- how to
- ecommerce
- dashboard
- ui
- credentials
- decentralized identity
- streams
- verify credential
---

# Dashboard

The Integration Services also includes a dashboard to view and manage all your identities, credentials and streams. The dashboard supports nearly every feature currently supported by the Audit Trail Gateway and SSI Bridge APIs. You can run the dashboard directly with the `npm run dev` script in the `./dashboard` directory. The dashboard gets automatically deployed if you chosen to install the Integration Services by docker-compose or Kubernetes.

## Login and Registration 

> This is the first page visible after opening the dashboard. An already created decentralized identity can be used to login. If no identity was created yet a new one can be registered by clicking on the *log in with DID* link.

Reviewer: ```suggestion
This is the first page visible after opening the dashboard. An already created decentralized identity can be used to login. If no identity was created yet a new one can be registered by clicking on the *register a new DID* link.
```
